Makefile.build

```diff
@@ -331,6 +331,13 @@ $(COQPP): $(COQPPCMO) coqpp/coqpp_main.ml
 	$(SHOW)'OCAMLC -a $@'
 	$(HIDE)$(OCAMLC) -I coqpp $^ -linkall -o $@
 
+###########################################################################
+# Specific rules for Uint63
+###########################################################################
+kernel/uint63.ml: kernel/write_uint63.ml kernel/uint63_x86.ml kernel/uint63_amd64.ml
+	$(SHOW)'WRITE $@'
+	$(HIDE)(cd kernel && ocaml unix.cma $(shell basename $<))
```

Comment:
I think it might be cleaner to do something similar to how coqide_os_specific is generated (add to configure to put the architecture info in config/Makefile, then make copies according to that) instead of writing a custom cp in ocaml. Not sure about how to control which file gets copied in dune though.

Reply:
@vbgl Would you mind taking care of this one? Thanks!

Reply:
Your request here seems contradictory with your request there: https://github.com/coq/coq/pull/6914#discussion_r237166603

What is the expected life-time of this `Makefile`?

Reply:
Can you say why they are contradictory? I haven't look at those request very precisely.

Reply:
There is the request from @ejgallego to have dune compile and run write_unint instead of just running it, and my request to stop using write_uint and just use a cp (in makefile) / `(copy#)` (in dune).

Reply:
Thanks for the clarification. So, what should we do?

Reply:
We also don't know how the architecture would be tested in Dune. What's the problem with the current approach?

Reply:
>We also don't know how the architecture would be tested in Dune. 

https://dune.readthedocs.io/en/latest/dune-files.html#variables-expansion `arch_sixtyfour`